<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Revolutions-Mac-Georgy.local" ignored="0" name="Surefire test" tests="7" failures="1" timestamp="15 Aug 2018 00:18:06 GMT" time="71.241" errors="0">
  <testcase name="afetchDataFromDialogflowTest on afetchDataFromDialogflowTest(tests.testPackage1.FBMessengerTests)" time="62.307" classname="tests.testPackage1.FBMessengerTests"/>
  <testcase name="verifyFAQBaggageTest on verifyFAQBaggageTest(tests.testPackage1.FBMessengerTests)" time="63.909" classname="tests.testPackage1.FBMessengerTests"/>
  <testcase name="verifyGetStartedTest on verifyGetStartedTest(tests.testPackage1.FBMessengerTests)" time="64.554" classname="tests.testPackage1.FBMessengerTests">
    <failure type="java.lang.AssertionError" message="TC Failed">
      <![CDATA[java.lang.AssertionError: TC Failed
at steps.Steps.FBMessengerSteps.verifyGetStarted(FBMessengerSteps.java:86)
at tests.testPackage1.FBMessengerTests.verifyGetStartedTest(FBMessengerTests.java:57)
]]>
    </failure>
  </testcase> <!-- verifyGetStartedTest on verifyGetStartedTest(tests.testPackage1.FBMessengerTests) -->
  <testcase name="getAll2TestDemo on getAll2TestDemo(tests.testPackage1.SecondSerenityTest)" time="0.358" classname="tests.testPackage1.SecondSerenityTest"/>
  <testcase name="getAll1Test on getAll1Test(tests.testPackage1.SecondSerenityTest)" time="0.822" classname="tests.testPackage1.SecondSerenityTest"/>
  <testcase name="getAll1Test on getAll1Test(tests.testPackage2.FirstSerenityTest)" time="0.343" classname="tests.testPackage2.FirstSerenityTest"/>
  <testcase name="getAll2Test on getAll2Test(tests.testPackage2.FirstSerenityTest)" time="0.797" classname="tests.testPackage2.FirstSerenityTest"/>
</testsuite> <!-- Surefire test -->
